### ISSUE TITLE
when installed with msm, all skills stop working.

### ISSUE THREAD

**Issue by ianni67** [forslund/fallback-aiml#3]
Something is wrong with this skill and my installation. Currently, if I install it with msm, all skills stop working (i.e. mycroft answers always with the same "can you rephrase" message) and I have to uninstall the skill.
I already tried re-starting mycroft after installation, with no luck.

**Comment by forslund** [comment]
Hi, thanks for reporting. I'll look into it. If possible can you upload your mycroft-skills.log to hastebin or similar service so I can see what's going on.

This skill should technically not do anything at the moment, it was disabled when a memory issue in mycroft-core together with the large RAM footprint of this skill caused trouble back in August.

**Comment by ianni67** [comment]
I'm sorry, but I could not locate the mycroft-skills.log file... -_- would you please give me a hint?

**Comment by forslund** [comment]
If it's on desktop (git clone) it's in mycroft-core/scripts/logs, if it's on a picroft or mark-1 it's in /var/log.
